infrastructure/terraform/accounts/s3.tf

```diff
@@ -0,0 +1,102 @@
+resource "aws_s3_bucket" "editorial_photography" {
+  bucket = "wellcomecollection-editorial-photography"
+  acl    = "private"
+
+  lifecycle_rule {
+    id      = "start_ia_move_to_glacier"
+    enabled = true
+    prefix  = ""
+
+    transition {
+      days          = 30
+      storage_class = "STANDARD_IA"
+    }
+
+    transition {
+      days          = 150
+      storage_class = "GLACIER"
+    }
+  }
+
+  versioning {
+    enabled = true
+  }
+
+  lifecycle {
+    prevent_destroy = true
+  }
+
+  lifecycle_rule {
+    enabled = true
+
+    noncurrent_version_expiration {
+      days = 30
+    }
+  }
+}
+
+resource "aws_s3_bucket_policy" "editorial_photography" {
+  bucket = aws_s3_bucket.editorial_photography.id
+  policy = data.aws_iam_policy_document.ep_bucket_policy.json
+}
+
+data "aws_iam_policy_document" "ep_bucket_policy" {
```

Comment:
Suggestion: call this `editorial_photography_bucket_policy`, both for consistency and because I think "EP" has a specific meaning in the context of editorial photography shoots. (e.g. shoots are labelled EP1234).